src/sys/signal.rs

```diff
@@ -3,21 +3,21 @@
 
 //! Operating system signals.
 
-use crate::{Error, Result};
```

Comment:
It looks like you ran rustfmt on this file.  Were there any content changes?  If not, please revert it.  If so, revert the formatting changes so we can see the content changes.

Reply:
Oops, yes, not intended.